# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1234974467 18000
#      Wed Feb 18 11:27:47 2009 -0500
# Node ID 35ea4f816bc95b9a7fa4c7dc1712165541f4e383
# Parent  061780d8da1e65d4cd66196ae036a0763ea7366a
ov-flt-cx-mat.cc, ov-cx-mat.cc: comment out unused static function

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,10 +1,13 @@
 2009-02-18  John W. Eaton  <jwe@octave.org>
 
+	* ov-cx-mat.cc (xabs): Comment out unused static function.
+	* ov-flt-cx-mat.cc (xabs): Ditto.
+
 	* c-file-ptr-stream.cc (c_file_ptr_buf::seekoff,
 	c_file_ptr_buf::seepos, c_zfile_ptr_buf::seekoff,
 	c_zfile_ptr_buf::seepos): Avoid unused paramter warnings.
 
 2009-02-18  Jaroslav Hajek  <highegg@gmail.com>
 
 	* data.cc (Fresize): Allow arbitrary number of parameters. Improve
 	documentation.
diff --git a/src/ov-cx-mat.cc b/src/ov-cx-mat.cc
--- a/src/ov-cx-mat.cc
+++ b/src/ov-cx-mat.cc
@@ -719,21 +719,23 @@ octave_complex_matrix::as_mxArray (void)
     {
       pr[i] = std::real (p[i]);
       pi[i] = std::imag (p[i]);
     }
 
   return retval;
 }
 
+#if 0
 static double
 xabs (const Complex& x)
 {
   return (xisinf (x.real ()) || xisinf (x.imag ())) ? octave_Inf : abs (x);
 }
+#endif
 
 static double
 ximag (const Complex& x)
 {
   return x.imag ();
 }
 
 static double
diff --git a/src/ov-flt-cx-mat.cc b/src/ov-flt-cx-mat.cc
--- a/src/ov-flt-cx-mat.cc
+++ b/src/ov-flt-cx-mat.cc
@@ -686,21 +686,23 @@ octave_float_complex_matrix::as_mxArray 
     {
       pr[i] = std::real (p[i]);
       pi[i] = std::imag (p[i]);
     }
 
   return retval;
 }
 
+#if 0
 static float
 xabs (const FloatComplex& x)
 {
   return (xisinf (x.real ()) || xisinf (x.imag ())) ? octave_Inf : abs (x);
 }
+#endif
 
 static float
 ximag (const FloatComplex& x)
 {
   return x.imag ();
 }
 
 static float
